src/models/backbones/efficientnet.py

```diff
@@ -0,0 +1,221 @@
+"""TorchOK EffcientNet.
+Adapted from https://github.com/rwightman/pytorch-image-models/blob/master/timm/models/efficientnet.py
+Copyright 2019 Ross Wightman
+Licensed under The Apache 2.0 License [see LICENSE for details]
+"""
+import math
+from typing import Union, Dict, Any, List
+
+import torch.nn as nn
+from torch import Tensor
+
+from src.constructor import BACKBONES
+from src.models.base_model import BaseModel
+from src.models.modules.bricks.convbnact import ConvBnAct
+from src.models.backbones.utils.utils import round_channels
+from src.models.backbones.utils.helpers import build_model_with_cfg
+from src.models.modules.blocks.inverted_residual import InvertedResidualBlock
+from src.models.backbones.utils.constants import IMAGENET_DEFAULT_STD, IMAGENET_DEFAULT_MEAN
+
+
+def _cfg(url='', **kwargs):
+    return {
+        'url': url, 'input_size': (3, 224, 224), 'pool_size': (7, 7),
+        'crop_pct': 0.875, 'interpolation': 'bicubic',
```

Comment:
Question about unused parameters crop_pct, interpilation. Are we need this parameters?